src/Symfony/Component/Form/Extension/Core/EventListener/ResizeFormListener.php

```diff
@@ -102,6 +102,10 @@ public function preSubmit(FormEvent $event)
         $form = $event->getForm();
         $data = $event->getData();
 
+        if ($data instanceof \Traversable) {
```

Comment:
@HeahDude it's ok in my last commit : 
https://github.com/ybensacq/symfony/commit/554307c6fcdf1ea9349caa0dc98a5fec9c047479